src/main/native/include/llvm/StringRef.h

```diff
@@ -76,7 +76,9 @@ namespace llvm {
     /// @{
 
     /// Construct an empty string ref.
-    /*implicit*/ StringRef() : Data(nullptr), Length(0) {}
+    /*implicit*/ StringRef() : Data(""), Length(0) {
+        set_null_terminated(true);
+      }
```

Comment:
Indentation?

Reply:
This actually matches the indentation for the other constructors.